src/main/java/bio/terra/cli/context/utils/Logger.java

```diff
@@ -0,0 +1,191 @@
+package bio.terra.cli.context.utils;
+
+import static ch.qos.logback.classic.Logger.ROOT_LOGGER_NAME;
+
+import bio.terra.cli.context.GlobalContext;
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.filter.ThresholdFilter;
+import ch.qos.logback.core.ConsoleAppender;
+import ch.qos.logback.core.OutputStreamAppender;
+import ch.qos.logback.core.rolling.FixedWindowRollingPolicy;
+import ch.qos.logback.core.rolling.RollingFileAppender;
+import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy;
+import ch.qos.logback.core.util.FileSize;
+import ch.qos.logback.core.util.StatusPrinter;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.File;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class to initialize and configure a Logback context and loggers for the main CLI app.
+ * Requires an already-loaded GlobalContext for defining logging output files and loading config
+ * properties.
+ */
+public class Logger {
+  private static final String LOG_FORMAT =
+      "%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n";
+
```

Comment:
Intuitively 50 MB * 5 files seems an order of magnitude high to me; maybe 5MB * 5?  I'll defer to your intuition though, as I don't know how big these get when debugging...

Reply:
I have no idea either. We may want to change this in the future once we actually start using these debug logs. Mostly I'm thinking of these logs as a hedge for when people other than us start installing this. Changed to 5MB * 5 instead.